### PR TITLE
chore: remove generated files before build

### PR DESCRIPTION
**Pull request by shuowu**
Currently deleted specs still sit in node SDK repo as generator script only keeps adding new files to the repo. We should run a clean script for generated files, then regenerate all based on provided specs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "main": "src/index.js",
   "scripts": {
     "banners": "./utils/maintain-banners.js",
+    "prebuild": "rm -rf ./src/models ./src/factories ./src/generated-client.js",
```
Comment (aarongranick-okta) [inline package.json:15]: We often use a module called `rimraf` for this. I'm guessing for cross-platform compatibility. Probably nobody is going to be running build on windows, but maybe a good practice to be cross-platform compatible.

**Comment by swiftone** [comment]
> We should run a clean script for generated files, then regenerate all based on provided specs

Do you have suggestions for such a script?

File: package.json (hunk 1 above)
Comment (shuowu) [reply package.json:15]: Will change, was just following another script in repo.

Comment (shuowu) [reply package.json]: Scripts updated for `prebuild` and `predocs`

**Pull request merged**